docs/DeveloperGuide.md

```diff
@@ -368,7 +370,7 @@ The `UI` component,
 
 <img src="images/KeyMapActivityDiagram.svg" style="width:80%;margin:0 10%">
 <div style="width:80%;margin:0 10%;text-align:center">
-    <b>Figure 4.3.1</b> Activity Diagram for UP and DOWN keys
+    <b>Figure 2.2</b> Activity Diagram for UP and DOWN keys
```

Comment:
Not updated as well

![image](https://user-images.githubusercontent.com/50812868/230290451-e17e6e3d-4225-4292-a0c7-3a2b57aaa847.png)

<img width="233" alt="image" src="https://user-images.githubusercontent.com/50812868/230290638-2b275f21-37df-4dfe-afb4-5e95de42fdd2.png">